acl/src/test/java/org/apache/rocketmq/acl/plain/PlainAccessValidatorTest.java

```diff
@@ -39,16 +27,21 @@
 import org.apache.rocketmq.common.protocol.heartbeat.HeartbeatData;
 import org.apache.rocketmq.common.protocol.heartbeat.ProducerData;
 import org.apache.rocketmq.common.protocol.heartbeat.SubscriptionData;
+import org.apache.rocketmq.remoting.exception.RemotingCommandException;
 import org.apache.rocketmq.remoting.protocol.RemotingCommand;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 
+import java.nio.ByteBuffer;
+import java.util.*;
```

Comment:
Same with last comment

Reply:
> Same with last comment

headerBuffer.capacity() - headerBuffer.position() is more precise than headerArray.length.
Howerver, headerBuffer.capacity() - headerBuffer.position() has more subtraction operation than headerArray.length. Considering performance, headerArray.length may be more appropriate.